packages/image_picker/image_picker_ios/example/ios/RunnerTests/PickerSaveImageToPathOperationTests.m

```diff
@@ -55,6 +55,76 @@ - (void)testSaveGIFImage API_AVAILABLE(ios(14)) {
   [self verifySavingImageWithPickerResult:result fullMetadata:YES];
 }
 
+- (void)testSaveBMPImage API_AVAILABLE(ios(14)) {
```

Comment:
Are you able to test the pre-iOS 14 codepath?

Reply:
Never mind, that code is dead, `FLTPHPickerSaveImageToPathOperation` is only available on iOS 14.